src/backend/yaas.js

```diff
@@ -1,16 +1,21 @@
 import React from 'react';
 
-export async function getDocument(docID) {
+var SERVER_URL = ""
```

Comment:
maybe set the default value to `var SERVER_URL = "https://yaas.azurewebsites.net/";`
and remove the `else` statement